pxtsim/svg.ts

```diff
@@ -73,15 +73,15 @@ namespace pxsim.svg {
 
 
     export function fill(el: SVGElement, c: string) {
-        (<SVGStylable><any>el).style.fill = c;
```

Comment:
No more SVGStyleable?

Reply:
Yeah, I couldn't find it in `lib.d.ts`. Either it's gone or the name changed.

Reply:
sniff :(